README.md

```diff
@@ -52,8 +52,11 @@ The following is an example of a valid ToMaR:
 ]
 ```
 
-The expression `.*` is used to represent the wildcard `#`, which in turn represents one or more levels in the MQTT topic hierarchy.
-The expression `([^/]+)` is used to represent the wildcard `+`, which in turn represents a single level in the MQTT topic hierarchy.
+- The expression `.*` is used to represent the wildcard `#`, which in turn represents one or more levels in the MQTT topic hierarchy.
+However, there are some cases that can lead to unexpected behavior when using the `.*` wildcard. Therefore, you should note the following:
+  - You can not use the `.*` wildcard in capturing groups. For example, the pattern `building/(\\w+)/room/(\\d{1,4})/(.*)` is invalid because it will capture the whole subtopic of the pattern, and `$3` placeholder might include `/` characters, which will lead to an invalid Kafka topic name.
```

Comment:
```suggestion
  - You cannot use the `.*` wildcard in capturing groups. 
  For example, the pattern `building/(\\w+)/room/(\\d{1,4})/(.*)` is invalid because it will capture the whole subtopic of the pattern, and `$3` placeholder might include `/` characters, which will lead to an invalid Kafka topic name.
```